examples/cdk/lib/example-function.Tracer.PatchAWSSDKv2.ts

```diff
@@ -3,11 +3,15 @@ import { Events } from '@aws-lambda-powertools/commons';
 import { Tracer } from '@aws-lambda-powertools/tracer';
 import { STS } from 'aws-sdk';
 
-// process.env.POWERTOOLS_SERVICE_NAME = 'tracerManualFn'; // Alternative to setting the service name in the constructor
 const tracer = new Tracer({ serviceName: 'tracerPatchAWSSDKv2Fn' });
-// To patch a specific AWS SDK Client, we pass it to the Tracer that will return an instrumented version of it
+// Alternatively, you can also set the service name using the POWERTOOLS_SERVICE_NAME environment variable
+// Learn more at: https://docs.aws.amazon.com/lambda/latest/dg/configuration-envvars.html
+
+// To patch a specific AWS SDK v2 Client, we pass it to the Tracer that will return an instrumented version of it
 const sts = tracer.captureAWSClient(new STS());
 
+// Here we are showing an example with manual instrumentation but you can do the same also with the captureLambdaHandler Middy Middleware and Class decorator
```

Comment:
(minor) I am not aware of this. O.o 

AFAI see, this is not mentioned in the documentation. Can we add it to the list in [this section](https://awslabs.github.io/aws-lambda-powertools-typescript/latest/core/tracer/#lambda-handler) . Another PR is fine.

Reply:
The actual example, in this file, is from L4 to L11 since this example wants to demonstrate how to instrument an AWS SDK v2 client. 

What I wanted to say with this comment is that even though this example uses manual instrumentation, _you don't need_ to use this method specifically over others (Middy middleware, decorator). Essentially, as long as you instrument the AWS SDK client with `tracer.capture*` it'll be traced & appear as custom subsegment on X-Ray no matter how you write your function. 

That's why [in the docs](https://awslabs.github.io/aws-lambda-powertools-typescript/latest/core/tracer/#patching-aws-sdk-clients) the AWS SDK patching example only has these lines.

I included a function in the example just because otherwise the lines above alone wouldn't run as a Lambda function and the lines from L15 onwards are similar to manual method in the section you linked.